CHANGELOG.md

```
@@ -1,54 +1,68 @@
-## 0.4.0-beta.13 (2026-03-14)
+# [0.5.1-beta.1](https://github.com/AntoniadisCorp/deepscrape/compare/v0.4.0-beta.13...v0.5.1-beta.1) (2026-03-15)
 
-* feat(codebase): harden functions env and admin credential loading ([327d4ab](https://github.com/AntoniadisCorp/deepscrape/commit/327d4ab))
-* ci(release): overhaul release workflow and expand copilot instructions ([8f81ab4](https://github.com/AntoniadisCorp/deepscrape/commit/8f81ab4))
-* chore: update changelog and package version [skip ci] ([6fa4954](https://github.com/AntoniadisCorp/deepscrape/commit/6fa4954))
-* chore(changelog): remove outdated entries for version 0.4.0-beta.12 ([4e8b80e](https://github.com/AntoniadisCorp/deepscrape/commit/4e8b80e))
+### Bug Fixes
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)